src/test/deploy/functions/services/auth.spec.ts

```diff
@@ -0,0 +1,495 @@
+import * as auth from "../../../../deploy/functions/services/auth";
+import * as backend from "../../../../deploy/functions/backend";
+import * as identityPlatform from "../../../../gcp/identityPlatform";
+import * as sinon from "sinon";
+import { expect } from "chai";
+import { BEFORE_CREATE_EVENT, BEFORE_SIGN_IN_EVENT } from "../../../../functions/events/v1";
+
+const BASE_EP = {
+  id: "id",
+  region: "us-east1",
+  project: "project",
+  entryPoint: "func",
+  runtime: "nodejs16",
+};
+
+const authBlockingService = new auth.AuthBlockingService();
+
+describe("authBlocking", () => {
+  let getConfig: sinon.SinonStub;
+  let setConfig: sinon.SinonStub;
+
+  beforeEach(() => {
+    getConfig = sinon
+      .stub(identityPlatform, "getBlockingFunctionsConfig")
+      .throws("Unexpected call to getBlockingFunctionsConfig");
```

Comment:
nit* for async functions, you'd use `reject`. I guess this works since `await` is forgiving 🤷🏼